components/PoweredBy/index.css

```diff
@@ -0,0 +1,9 @@
+.as-powered-by--block {
+  text-align: right;
+  line-height: 1.6em;
+}
+
+.as-powered-by--block img {
```

Comment:
If we got the BEM way, we should not use selectors that target tags, but only classes. You should add a class of  `.as-powered-by--img` on the `img` itself.